components/datetime/tests/patterns/tests/time_zones.json

```diff
@@ -652,7 +652,7 @@
           "GenericNonLocationShort"
         ],
         "fallback_formats": ["LocalizedGmt"],
-        "expected": ["GMT"]
+        "expected": ["Тролль"]
```

Comment:
The changes in this file _appear_ to be "actual bugs" based on my reading of the spec and the locale data.